docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,162 @@
+# Publish GOV.UK Frontend when `main` branch has unreleasable changes
+
+Do not use this document to release changes for previous major versions. We have not tested the document in this scenario, and extra work would be needed to tell NPM not to mark the release as the 'latest'
+
+Follow this document to publish a patch release of GOV.UK Frontend when the `main` branch contains unreleasable changes. For example, you might need to release a fix:
+
+- as part of a patch release, after the team has started to merge changes for a new feature release into the `main` branch - for example, a 3.14.x release once we've started merging changes for 3.15.0
+- as part of a release, after the team has started to merge changes for a new breaking release into the `main` branch - for example, a 3x.x release once we've started merging changes for 4.0.0
+
+If the `main` branch contains unreleased changes that are releasable immediately, [follow the steps in Publishing GOV.UK Frontend](/docs/releasing/publishing.md) instead.
+
+If the `main` branch only has a few unreleasable changes and these are easy to revert, you can do the following:
+
+1. Revert the unreleasable changes on the `main` branch
+2. Follow the steps to publish GOV.UK Frontend
+3. Add the reverted changes to the `main` branch
+
```

Comment:
Suggest the following changes to minimise the passive, be more clear about what you are achieving here, remove the need for the colon as recommended in the style guide, and make sure the numbered bullets are in the right format (complete sentences) as per the style guide:

"If the `main` branch only has a few unreleasable changes, you can temporarily revert those changes.

1. Revert the unreleasable changes on the `main` branch.
1. Publish GOV.UK Frontend.
1. Add the reverted changes back into the `main` branch."

Reply:
Thanks @jonathanglassman - actioned.